<launch>
  <arg name="manager" default="visual_altimeter_manager" />
  <arg name="standalone" default="false" />
  <arg name="stereo" />
  <arg name="show_image" default="false" />

  <node if="$(arg standalone)" name="$(arg manager)" pkg="nodelet" type="nodelet" args="manager" output="screen" />

  <node name="stereo_feature_extractor" pkg="nodelet" type="nodelet" args="load stereo_feature_extraction/StereoFeatureExtractor $(arg manager)" output="screen" >
    <remap from="stereo" to="$(arg stereo)" />
    <param name="show_image" value="$(arg show_image)" />
    <rosparam file="$(find stereo_feature_extraction)/cfg/stereo_feature_extractor_params.yaml" />
  </node>

  <node name="plane_fitting_node" pkg="stereo_feature_extraction" type="plane_fitting_node" output="screen">
    <remap from="point_cloud" to="stereo_feature_extractor/point_cloud" />
  </node>

</launch>
